assets/stylesheets/sections/gutenberg-editor.scss

```diff
@@ -21,35 +21,12 @@
 // NUX
 @import '../../../node_modules/@wordpress/nux/build-style/style';
 
+// Edit-post component styles
+@import '../../../node_modules/@wordpress/edit-post/build-style/style';
```

Comment:
My concern with this is that `edit-post` components will be pinned to old version (at the time when they were copied) while their styles will keep getting updated and possibly diverge over time. This can cause regressions whenever `edit-post` package is updated by renovate PRs (e.g. some `classname` could change, which wouldn't be reflected in related component code). I'm not strictly against it, we just have to take into account additional testing and maintenance burden this will introduce for future package updates.

Reply:
Oh yes. I guess we can then also import the components from the `@wordpress/edit-post` package, but probably not the scope of this, so I'm going to revert it.

Maybe we can take a look at it on #26642